standard/expressions.md

```diff
@@ -5984,6 +6069,27 @@ When a property or indexer declared in a *struct_type* is the target of an assig
 >
 > *end example*
 
+### §deconstructing-assignment-new-clause Deconstructing assignment
+
+If the left operand of a `=` operator is classified as a tuple, the right hand side is ***deconstructed*** into individual expressions, which are assigned to each of the elements of the left hand side tuple.
+
+If any of the tuple elements of the left operand have an identifier, a compile-time error occurs. If any of the tuple elements of the left operand is a declaration expression, and any other element is not a declaration expression or discard, a compile-time error occurs.
+
+The evaluation of a deconstructing assignment `(R1, ..., Rn) = E` proceeds as follows:
```

Comment:
Is there a reason for picking R here? Could we use L instead, given that it's on the left side of the operator?

Reply:
Changed naming in the new formulation - they are now `x1...xn` and `y1...yn` respectively, to match existing naming in the simple assignment section.